v3/src/components/axis/components/axis.tsx

```diff
@@ -59,36 +60,35 @@ export const Axis = ({
   const {parentElt, wrapperElt, setWrapperElt} = useAxisBoundsProvider(place, parentSelector)
 
   useAxis({
-    axisModel, axisElt, label, enableAnimation, showScatterPlotGridLines, centerCategoryLabels,
-    titleRef
+    axisModel, axisElt, titleRef, axisTitle: label, centerCategoryLabels
   })
 
   useDropHandler(droppableId, active => {
     const droppedAttrId = getDragAttributeId(active)
     droppedAttrId && isDropAllowed(place, droppedAttrId) && onDropAttribute?.(place, droppedAttrId)
   })
 
-  const [xMin, xMax] = scale?.range() || [0, 100]
-  const halfRange = Math.abs(xMax - xMin) / 2
-  useEffect(function setupTitle() {
-    select(titleRef.current)
-      .selectAll('text.axis-title')
-      .data([1])
-      .join(
-        // @ts-expect-error void => Selection
-        (enter) => {
-          enter.append('text')
-            .attr('class', 'axis-title')
-            .attr('text-anchor', 'middle')
-            .attr('data-testid', `axis-title-${place}`)
-        })
-
-  }, [axisElt, halfRange, label, place])
+  const getSubAxes = () => {
+    const numRepetitions = dataConfiguration?.numRepetitionsForPlace(place) ?? 1
+    layout.getAxisScale?.(place).setRepetitions(numRepetitions)
```

Comment:
Should not be doing model synchronization in the render path of the Axis component. Addressed in my code review commit.